tests/config/hsc-config.py

```diff
@@ -1,2 +1,3 @@
 # no colorterms in tests unless specifically requested (this overrides the default HSC config)
 config.applyColorTerms = False
+config.astrometryOutlierRelativeTolerance = False
```

Comment:
The docs have this as type `float`, not `bool`, so I believe this should be set to 0.  Also, you need a carriage return.

I found the commit message a tad confusing as you are not actually removing the use of `obs_subaru` here.